configure.bat

```diff
@@ -28,19 +28,15 @@ exit /B %errorlevel%
     )
     set envfile=
 
-    set all_set=1
-    if not defined OPENSSL_DIR set all_set=0
-
-    if %all_set%==1 (
-        @echo - Using OpenSSL: %OPENSSL_DIR%
-    ) else (
-        @echo At least one of the following variables were not set:
-        @echo     - OPENSSL_DIR: %OPENSSL_DIR%
-        @echo Please define them using by creating a "env.bat" file containing:
-        @echo     @set OPENSSL_DIR=^<your OpenSSL directory^>
-        exit /B 1
+    if not defined openssl_dir (
+        echo [INFO] "OPENSSL_DIR" environment variable is not defined. Relying
+        echo on meson to properly find OpenSSL with standard
+        echo `dependency('openssl'^)` command. If meson does not find OpenSSL,
+        echo consider setting it to your current OPENSSL library directory. In
+        echo this case, you may consider creating a env.bat file containing:
+        echo
+        echo     @set OPENSSL_DIR=^<your OpenSSL directory^>
```

Comment:
Much better, thanks.

Reply:
those `^` should really be here?

Reply:
This is windows' `\` for escaping characters :D So yeah, they should. Explaining the excapings:

- `)` is considered a closing scope, `^)` is a literal ")";
- `<` and `>` file input/output redirs, `^<` and `^>` are literal lt and gt.

Reply:
windows.....